firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -140,6 +140,8 @@ expect class DocumentReference {
     suspend fun update(vararg fieldsAndValues: Pair<FieldPath, Any?>)
 
     suspend fun delete()
+
+    companion object
```

Comment:
Whats the point of this empty companion object (non of the actuals implement anything for it)